systers_portal/users/forms.py

```diff
@@ -36,3 +38,16 @@ class SystersUserForm(forms.ModelForm):
     class Meta:
         model = SystersUser
         fields = ('country', 'blog_url', 'homepage_url', 'profile_picture')
+
+
+class SystersChangePasswordForm(ChangePasswordForm):
+    # The clean_password1() method which checks that new password differs from the old one
+    # is added to allauth ChangePasswordForm class.
+    # So ChangePasswordForm class is derived.
+    def __init__(self, *args, **kwargs):
+        super(SystersChangePasswordForm, self).__init__(*args, **kwargs)
+
+    def clean_password1(self):
+        if self.cleaned_data["password1"] == self.cleaned_data.get("oldpassword", None):
+            raise ValidationError("New password must differ from the old one.")
+        return self.cleaned_data["password1"]
```

Comment:
Added an extra commit to change few variable names here and merged w/ 3d870165ad0b93e55c084a2e463bede642ec6ff3.